src/commons/application/ApplicationTypes.ts

```diff
@@ -302,7 +302,18 @@ export const defaultSession: SessionState = {
     collectibles: {}
   },
   xp: 0,
-  notifications: []
+  notifications: [],
+  githubLoginID: '',
+  githubName: '',
+  githubEmail: '',
+  userRepos: [],
+  githubRepositoryFilepath: '',
+  githubSaveMode: '',
+  githubRepositoryName: '',
+  githubCommitMessage: '',
+  pickerType: '',
+  isPickerOpen: false,
```

Comment:
can also check with @angelsl if all these are needed in the Redux store (especially the isPickerOpen and isDialogOpen) as i think there are plans to trim down on the store and remove state that is local to the workspace #1555.

and also, im assuming these are placed under `defaultSession` instead of `defaultPlayground` as there are plans to make the Github login persist through refreshes and across workspaces